package.json

```diff
@@ -32,7 +34,7 @@
   },
   "files": [
     "/bin",
-    "/lib",
+    "/dist",
```

Comment:
why this change?

Reply:
Just to make it more semantic about what is being built. This repo doesn't represent a library.

Reply:
@RodEsp Are you concerned that this is a divergence from every other oclif plugin and library? Or are you thinking we'd update all of those as well?

Reply:
I'm not opposed to the change - I just think that following a single less-than-ideal pattern is better than following multiple patterns.

Do you mind logging a WI for updating all the other repos so that we don't loose track of it?